src/Discovery/Discovery.tsx

```diff
@@ -89,7 +89,7 @@ export const renderFieldContent = (content: any, contentType: 'string'|'paragrap
     if (Array.isArray(content)) {
       return content.join(', ');
     }
-    return content.toLocaleString();
+    return content ? content.toLocaleString() : 'N/A';
```

Comment:
nice I like this as this won't crash undef values

Reply:
Two questions: does a missing `content` value need to be type matched? eg: if an int is missing the portal code should be looking for 0 as opposed to `N/A`? also how about `__manifest`? If there is none, that should be [] as opposed to `N/A`?

Reply:
Good catch. I have updated the expression